README.md

```diff
@@ -39,6 +43,107 @@ From the root of the cloned project, enter the following in your command line to
 npm ci
 ```
 
+## Content Management System (CMS)
+
+[Netlify CMS](https://netlifycms.org) is a client-side React application which manages files in a git repository,
+creating pull requests when new content is drafted and merging them when it is published. Access to this website's
+CMS is managed via [Netlify Identity](https://docs.netlify.com/visitor-access/identity/). If you need access to the
+CMS, a team administrator must invite you to create a Netlify Identity account.
+
+### CMS Configuration
+
+The CMS is configured via a [config.yml](https://github.com/inclusive-design/wecount.inclusivedesign.ca/blob/e082fdd17c08d53fd6910f055132e3dd150fbb79/src/admin/config.yml)
+file according to Netlify CMS' [specifications](https://www.netlifycms.org/docs/configuration-options/).
+As an example, here is the configuration for the workshops collection, stored in [`src/collections/workshops`](src/collections/workshops):
+
+```yaml
+  - name: workshops
+    label: Workshops
+    label_singular: Workshop
+    folder: "src/collections/workshops"
+    slug: "{{title}}"
+    create: true
+    fields:
+      - label: "Event Title"
+        name: "title"
+        widget: "string"
+      - label: "Event ID"
+        name: "id"
+        widget: "uuid"
+        hint: "The ID is used to associate comments with this workshop and cannot be edited."
+      - label: "Permanent Link"
+        name: "permalink"
+        widget: "string"
+        required: false
+        hint: |-
+          If you do not specify a permanent link, one will be automatically generated from the event title.
+          Permalinks must have the format /initiatives/event-title/ (the trailing slash is required).
```

Comment:
Shall the format of permalinks be /initiatives/event-id/? event-title -> event-id

The same comment applies to `src/admin/config.yml` where the hint for the "permalink" field is defined.

Reply:
See comment below: https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/468#discussion_r605910729